src/Moryx.Resources.Management/ModuleController/ModuleController.cs

```diff
@@ -105,16 +115,16 @@ protected override void OnStop()
             resourceManager.Stop();
         }
 
-        #endregion
+#endregion
 
-        #region FacadeContainer
+#region FacadeContainer
```

Comment:
The default resharper setting is that the region definition is on the same level with the first line of code of the region. Compiler definitions (like if, else...) are starting at the beginning.

Reply:
Sorry, Studio or ReSharper AutoFormatted that when inserting the Region.

Reply:
I took a second look. That whole region is overblown. The class has 112 LOC.